spec/visual/app.rb

```diff
@@ -46,13 +46,15 @@ def query_string
 
     # Invoke a specific formatter based on what one intends to test
     @formatter = params[:inline] ? formatters::HTMLInline.new(@theme) : formatters::HTML.new
-    @formatter = if params[:line_numbers] == 'true'
-                   if params[:line_table] == 'false'
+
+    if params[:line_numbers] == 'true'
```

Comment:
we usually accept `'1'` as an input here. I'd just check for non-nil and `!= 'false'`

Reply:
and `!= '0'` i suppose

Reply:
Checking for non-nil or `!= 'false'` would mean that `line_numbers` params are enabled by default. I don't think that's what you intend to have.. is it?

Reply:
is it not nil by default?

Reply:
Yes it is..
```suggestion
    if params[:line_numbers] && params[:line_numbers] != 'false'
```

Reply:
Can I introduce a separate module just to be used by the visual test app?
It would have the above logic and a variant of the following from `Rouge::Lexer`:
```ruby
    def as_bool(val)
      case val
      when nil, false, 0, '0', 'off'
        false
      when Array
        val.empty? ? true : as_bool(val.last)
      else
        true
      end
    end
```

Reply:
Sure, you can also just define private methods in the visual test app.